packages/fxa-settings/src/lib/file-utils-legal.tsx

```diff
@@ -2,74 +2,42 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { determineLocale } from 'fxa-shared/l10n/determineLocale';
-import sentryMetrics from 'fxa-shared/lib/sentry';
+import { GET_LEGAL_DOC } from '../models';
+import { ApolloClient } from '@apollo/client';
 
 export enum LegalDocFile {
   privacy = 'firefox_privacy_notice',
   terms = 'firefox_cloud_services_tos',
 }
 
-const LEGAL_DOCS_PATH = '/settings/legal-docs';
-
-// TODO: probably move this + clone script to gql-api to reduce network requests
-
-const fetchLegalMdByLocale = async (locale: string, file: LegalDocFile) => {
-  // use the general app error for now, might change when moving this to gql-api
-  const error = 'Something went wrong. Please try again later.';
-
-  try {
-    const response = await fetch(`${LEGAL_DOCS_PATH}/${locale}/${file}.md`);
-    if (response.ok) {
-      const resText = await response.text();
-      if (resText.includes('<!DOCTYPE html>')) {
-        return { error };
-      }
-      return { markdown: resText };
-    } else {
-      throw Error(response.statusText);
-    }
-  } catch (e) {
-    sentryMetrics.captureException(e);
-
-    // TODO: If the first preferred language can't be loaded, try
-    // the next 2? and then fallback to English + clean this up
-    if (locale !== 'en') {
-      try {
-        const response = await fetch(`${LEGAL_DOCS_PATH}/en/${file}.md`);
-        if (response.ok) {
-          const resText = await response.text();
-          if (resText.includes('<!DOCTYPE html>')) {
-            return { error };
-          }
-          return { markdown: resText };
-        }
-      } catch (e) {
-        sentryMetrics.captureException(e);
-        return { error };
-      }
-    }
-    return { error };
+export const fetchLegalMd = async (
+  apolloClient: ApolloClient<object> | undefined,
+  locale: string,
+  file: string
+): Promise<{
+  markdown?: string;
+  error?: string;
+}> => {
+  const error = `Something went wrong. Try again later.`;
```

Comment:
I could have sworn this string already existed in the codebase localized, but now I don't see it. We could probably use a 1-pointer in the cleanup epic to review this / localize it 🤔

Reply:
I also looked but didn't see it. I think there's a couple cases of this, so a followup would be good.

Reply:
I found these similar-adjacent strings (with "Please" prefix), but neither seem to be in fxa-settings .ftl files:

```ini
git grep -in "something went wrong" packages/**/*.ftl | cat

packages/fxa-payments-server/src/lib/en.ftl:6:basic-error-message = Something went wrong. Please try again later.
packages/fxa-react/components/AppErrorDialog/en.ftl:4:app-general-err-message = Something went wrong. Please try again later.
packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/en.ftl:25:confirm-signup-code-error-message = Something went wrong. A new code could not be sent.
```

Reply:
Follow up filed.